roles/nf-core/defaults/main.yml

```diff
@@ -20,13 +20,15 @@ pipelines:
     release: "{{ sarek_tag }}"
     container_dir: "{{ biocontainers_dirname }}"
     pipeline_specific_containers:
-      - sarekvep
-      - sareksnpeff
+      - vep-106.1
+      - snpeff-5.1
```

Comment:
Do you know how these versions ☝️ relate to the sarek version? I would be inclined to say that these should not be changed right now either. And neither should the addition of reference genomes below 👇

Reply:
Good catch! I excluded this commit now, too.